dev-libs/websocketpp/websocketpp-0.8.1.recipe

```diff
@@ -0,0 +1,48 @@
+SUMMARY="Websocket++ header-only library"
+DESCRIPTION="WebSocket++ is a header only C++ library that implements RFC6455 \
+The WebSocket Protocol. It allows integrating WebSocket client and server \
+functionality into C++ programs. It uses interchangeable network transport \
+modules including one based on raw char buffers, one based on C++ iostreams, \
+and one based on Asio (either via Boost or standalone). End users can write \
+additional transport policies to support other networking or event libraries as needed."
+HOMEPAGE="https://github.com/zaphoyd/websocketpp"
+SOURCE_URI="https://github.com/zaphoyd/websocketpp/archive/0.8.1.zip"
+PATCHES="websocketpp-0.8.1.patch"
+REVISION="1"
+LICENSE="BSD (2-clause)"
+COPYRIGHT="2014, Peter Thorson."
+CHECKSUM_SHA256="80f49469f1702ec64ef5d8e71302f6b781dca8ae06e560421cab42961c8c7ce6"
```

Comment:
This is still not in the right order. See https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering again.